web-admin/src/main/java/pl/edu/icm/unity/webadmin/tprofile/TranslationProfilesComponent.java

```diff
@@ -438,6 +441,30 @@ public boolean handleProfile(TranslationProfile profile)
 			dialog.show();
 		}
 	}
+	
+	private class ExportActionHandler extends SingleActionHandler
+	{
+		public ExportActionHandler()
+		{
+			super(msg.getMessage("TranslationProfilesComponent.exportAction"), Images.save.getResource());
+		}
+
+		@Override
+		public void handleAction(Object sender, final Object target)
+		{
+			SimpleFileDownloader downloader = new SimpleFileDownloader();
+			addExtension(downloader);
+			@SuppressWarnings("unchecked")
+			GenericItem<TranslationProfile> item = (GenericItem<TranslationProfile>) target;
+			final StreamResource resource = new StreamResource(() -> {
+				return new ByteArrayInputStream(item.getElement().toJsonObject().toString().getBytes());
```

Comment:
2 things here: use ObjectMapper to obtain String value, and ensure that UTF encoding is used when generating byte representation.